k8s/namespaces/ia/ia-case-api/perftest.yaml

```diff
@@ -2,8 +2,11 @@ apiVersion: helm.fluxcd.io/v1
 kind: HelmRelease
 metadata:
   name: ia-case-api
+  namespace: ia
```

Comment:
you defo don't need this in the perftest.yaml, and shouldn't need it in the base file